Binary_Search/RotationCount.java

```diff
@@ -0,0 +1,71 @@
+import java.util.Scanner;
+
+class RotationCount
+{
+
+	static int findRotations(int[] arr)
```

Comment:
Please use 4 spaces instead of 1 tab for indentation.

Reply:
Sure! I will make the changes.